arch/arm64/boot/dts/qcom/msm8917-motorola-nora.dts

```diff
@@ -147,6 +147,108 @@
 	status = "okay";
 };
 
+&camss {
+	vdda-supply = <&pm8937_s3>;
```

Comment:
are you sure this is `s3` and not `l2`?

Reply:
oops. actually not sure where I got `s3` from at all, lol. it should be `l2`

Reply:
8953 have s3